### PR TITLE
CU-86a0yz7tw - Add calculateFee method on Neo3Invoker to be used on W…

### PR DESCRIPTION
**Pull request by endkeyCoder**
…cSdk

### CODE REVIEW

**Comment by melanke** [comment]
Task linked: [CU-86a0yz7tw Add calculateFee method on Neo3Invoker to be used on WcSdk](https://app.clickup.com/t/86a0yz7tw)

File: common/changes/@cityofzion/neon-dappkit-types/CU-86a0yz7tw-2_2023-09-29-20-41.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cityofzion/neon-dappkit-types",
+      "comment": "It has been exported, the 'calculateFee' function in the Neo3Invoker interface.",
```
Comment (melanke) [inline common/changes/@cityofzion/neon-dappkit-types/CU-86a0yz7tw-2_2023-09-29-20-41.json:5]: ALWAYS use Google translate to write these sentences

File: common/changes/@cityofzion/neon-dappkit/CU-86a0yz7tw-2_2023-09-29-20-41.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cityofzion/neon-dappkit",
+      "comment": "Now, the 'calculateFee' function returns a string in the 'networkFee' and 'systemFee' properties.",
```
Comment (melanke) [inline common/changes/@cityofzion/neon-dappkit/CU-86a0yz7tw-2_2023-09-29-20-41.json:5]: here too

**Review by melanke — CHANGES_REQUESTED** [review]

**Comment by endkeyCoder** [comment]
can you close this PR? The same solution was implemented here -> https://github.com/CityOfZion/neon-dappkit/pull/20

**Pull request closed without merging**